### PR TITLE
Show backtraces in case of unit test failure.

### PR DESCRIPTION
**Pull request by nical**
There is probably a more elegant way to do this, I just prefixed with 'RUST_BACKTRACE=1' the ```cargo test``` commands I came across.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1268)

### CODE REVIEW

**Comment by kvark** [comment]
@nical it would be simple to do
```yaml
env:
  - RUST_BACKTRACE=1
```
Example - https://github.com/slide-rs/hibitset/pull/4/files#diff-354f30a63fb0907d4ad57269548329e3R15

**Comment by nical** [comment]
My limited understanding of travis is that adding a ``` - RUST_BACKTRACE=1``` item in the env section will add another combination of tests to run in the test matrix. I don't see an example where a single item has several environment variables set like ``` - BUILD_KIND=DEBUG, RUST_BACKTRACE=1```, maybe I should try and see.

File: .travis.yml [hunk 1]
```diff
@@ -24,9 +24,9 @@ before_install:
   - pip install mako servo-tidy
 script:
   - servo-tidy
-  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender_traits && cargo test --verbose --features "ipc"); fi
-  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender_traits && cargo test --verbose); fi
+  - if [ $BUILD_KIND = DEBUG ]; then (cd webrender_traits && RUST_BACKTRACE=1 cargo test --verbose --features "ipc"); fi
```
Comment (kvark) [inline .travis.yml]: let's remove `RUST_BACKTRACE=1` from here, given that `env` appears to be working?

Comment (nical) [inline .travis.yml]: oops!

**Comment by nical** [comment]
This version should [work](https://docs.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item) more elegantly.

**Pull request merged**